app/views/registeredAddressForEori.scala.html

```diff
@@ -0,0 +1,75 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import controllers.routes._
+@import models.{Mode, RegisteredAddressForEori}
+@import views.ViewUtils._
+
+@(appConfig: FrontendAppConfig, form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("registeredAddressForEori.title")}",
+    appConfig = appConfig,
+    bodyClasses = None) {
+
+    @helpers.form(action = RegisteredAddressForEoriController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("registeredAddressForEori.heading")
+
+        <ul role="list" class="govuk-check-your-answers cya-questions-long section">
+            <li>
+                <div class="cya-question">EORI number</div>
+                <div class="cya-answer">GB6723190</div>
```

Comment:
Is this meant to be hard-coded?

Reply:
Currently this isn't dynamic data. I can add this to messages, otherwise this is as expected as far as I understand.